src/__test__/components/Search/SearchControls.test.tsx

```diff
@@ -1,19 +1,26 @@
 import { render, screen } from '@testing-library/react';
-import { describe, it, expect } from 'vitest';
+import { describe, it, expect, beforeEach } from 'vitest';
 import { SearchControls } from '../../../components/SearchPanel/Search/SearchControls';
 import userEvent from '@testing-library/user-event';
 import { PokemonListContext } from '../../../types/contexts';
 import { contextMock } from '../../mocks/data';
 import { MockProvider } from '../../mocks/MockProvider';
+import type { PokemonListContextProps } from '../../../types/interfaces';
 
 describe('SearchControls component', () => {
+  let context: PokemonListContextProps;
+
+  beforeEach(() => {
+    context = { ...contextMock };
+  });
+
   it('renders with form elements', () => {
-    contextMock.query = '';
-    contextMock.loading = false;
+    context.query = '';
```

Comment:
for such cases you can create new object by destructing contextMock and change some values that you have to change
do it in scope of appropriate test